CMakeLists.txt

```diff
@@ -138,14 +138,21 @@ foreach(link ${content_links})
 endforeach()
 ExternalData_Add_Target(ITKSphinxExamplesData)
 
-if(${ITK_WRAP_PYTHON})
-  # Use the same Python interpreter as ITK to run the tests.
-  find_package(PythonInterp ${ITK_WRAP_PYTHON_VERSION} REQUIRED)
-  mark_as_advanced(PYTHON_EXECUTABLE)
+set(PYTHON_REQUIRED_VERSION 3.6)
+set(Python_ADDITIONAL_VERSIONS 3.12 3.11 3.10 3.9 3.8 3.7 3.6)
```

Comment:
Python 3.6 reached end of life in December and is no longer supported. It should be removed from this list.